source/tutorials/kata.rst

```diff
@@ -37,89 +37,90 @@ Install Kata Containers
 ***********************
 
 Kata Containers is included in the :file:`containers-virt` bundle.
-To install the framework, enter the following command:
+To install the framework:
 
-.. code-block:: bash
+#. Install the containers-virt bundle:
 
-   sudo swupd bundle-add containers-virt
+   .. code-block:: bash
 
-Restart the Docker\* and Kata Containers systemd services.
+      sudo swupd bundle-add containers-virt
 
-.. code-block:: bash
+#. Reload and restart the Docker\* systemd service.
 
-   sudo systemctl daemon-reload
-   sudo systemctl restart docker
+   .. code-block:: bash
+
+      sudo systemctl daemon-reload
+      sudo systemctl restart docker
 
 Run Kata Containers
 *******************
 
+To use kata as the runtime for an individual container, add
+:command:`--runtime=kata-runtime` to the :command:`docker run` command. For
+example:
+
 .. code-block:: bash
 
-   sudo docker run -ti busybox sh
+   sudo docker run --runtime=kata-runtime -ti busybox sh
 
-.. note::
+To use kata as the default runtime for all Docker containers:
 
-   If you use a proxy server and your proxy environment variables are already
-   set, run the following commands as a shell script to configure Docker:
+#. Create a systemd drop-in configuration for Docker:
 
    .. code-block:: bash
 
-      docker_service_dir="/etc/systemd/system/docker.service.d/"
-      sudo mkdir -p "$docker_service_dir"
-      cat <<EOF | sudo tee "$docker_service_dir/proxy.conf"
+      sudo mkdir -p /etc/systemd/system/docker.service.d/
```

Comment:
We might want to consider doing this step using /etc/docker/daemon.json to set the default runtime.  It is a tough call because we generally use systemd units but `kata-deploy` uses `/etc/docker/daemon.json` to add its runtimes.   I don't think you have to do a daemon-reload when using dockers daemon.json.

How to do this is at https://docs.docker.com/engine/reference/commandline/dockerd/

```
	"no-new-privileges": false,
	"default-runtime": "runc",
	"oom-score-adjust": -500,
	"node-generic-resources": ["NVIDIA-GPU=UUID1", "NVIDIA-GPU=UUID2"],
	"runtimes": {
		"cc-runtime": {
			"path": "/usr/bin/cc-runtime"
		},
		"custom": {
			"path": "/usr/local/bin/my-runc-replacement",
			"runtimeArgs": [
				"--debug"
			]
		}
	},
	"default-address-pools":[
		{"base":"172.80.0.0/16","size":24},
		{"base":"172.90.0.0/16","size":24}
	]
}
```

Reply:
That's a really good point. I will add a note about the `daemon.json` method but leave the CLI method as the primary in the steps.